docs-2.0/nebula-operator/4.connect-to-nebula-graph-service.md

```diff
@@ -0,0 +1,142 @@
+# Connect to Nebula Graph databases with Nebular Operator
+
+After creating a Nebula Graph cluster with Nebula Operator on Kubernetes, you can connect to Nebula Graph databases from within the cluster and outside the cluster.
+
+## Prerequisites
+
+Create a Nebula Graph cluster with Nebula Operator on Kubernetes. For more information, see [Deploy Nebula Graph clusters with Kubectl](3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md) or [Deploy Nebula Graph clusters with Kubectl](3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md).
+
+## Connect to Nebula Graph databases from within a Nebula Graph cluster
+
+When a Nebula Graph cluster is created, Nebula Operator automatically creates a Service named `<cluster-name>-graphd-svc` with the type `ClusterIP` under the same namespace. With the IP of the Service and the port number of the Nebula Graph database, you can connect to the Nebula Graph database.
+
+1. Run the following command to check the IP of the Service:
+
+  ```bash
+  $ kubectl get service -l app.kubernetes.io/cluster=<nebula>  #<nebula> is a variable value. Replace it with the desired name.
+  NAME                       TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)                                          AGE
+  nebula-graphd-svc          ClusterIP   10.98.213.34   <none>        9669/TCP,19669/TCP,19670/TCP                     23h
+  nebula-metad-headless      ClusterIP   None           <none>        9559/TCP,19559/TCP,19560/TCP                     23h
+  nebula-storaged-headless   ClusterIP   None           <none>        9779/TCP,19779/TCP,19780/TCP,9778/TCP            23h
+  ```
+
+  Services of the `ClusterIP` type only can be accessed by other applications in a cluster. For more information, see [ClusterIP](https://kubernetes.io/docs/concepts/services-networking/service/).
+
+2. Run the following command to connect to the Nebula Graph database using the IP of the `<cluster-name>-graphd-svc` Service above:
```

Comment:
Why `Service`

Reply:
Service is a specified resource object in k8s，we use keep the first letter capitalized.